packages/x-components/src/views/design-system/design-system-tag.vue

```diff
@@ -143,15 +143,38 @@
         <span>text tag extra large name</span>
       </div>
     </article>
+    <article class="x-list x-list--wrap x-list--gap-06 x-list--align-start">
```

Comment:
Im not sure about this example, I wouldn't add it or mix tags and curated with different styles

Reply:
IMO having them alone is enough.